types/validator/index.d.ts

```diff
@@ -950,6 +950,7 @@ declare namespace validator {
          * @default false
          */
         no_symbols?: boolean | undefined;
+        locale?: AlphaLocale;
```

Comment:
```suggestion
        locale?: AlphaLocale | undefined;
```
just to stick to the rest (TS 4.4 exact optional types compat: https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#exact-optional-property-types)